modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -9,17 +9,19 @@ The {product-title} cluster log forwarding feature uses a combination of _output
 
 [NOTE]
 ====
-If you want to use only the default internal {product-title} Elasticsearch instance, do not configure any outputs and pipelines. 
+If you want to use only the default internal {product-title} Elasticsearch logstore, do not configure any outputs and pipelines. 
 ==== 
 
-An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+By default, the Cluster Logging Operator creates `logforwarding` configuration file that forwards logs to the default internal Elasticsearch logstore. To use the log forwarding feature, you create a custom `logforwarding` configuration file to send logs to endpoints you specify. In this configuration file, the `disableDefaultForwarding=true` parameter disables the default forwarding behavior. This change assumes you define outputs and pipelines to forward logs for storage. 
```

Comment:
Could we remove the disableDefaultForwarding  here?

Reply:
Yes. See https://github.com/openshift/openshift-docs/pull/21106#discussion_r413353866

Reply:
I don't like the word '`logforwarding` configuration file '  in "The Cluster Logging Operator creates `logforwarding` configuration file that forwards logs to the default internal Elasticsearch logstore".  as we couldn't find a file named logforwarding.